src/EnergyPlus/HighTempRadiantSystem.cc

```diff
@@ -1422,22 +1416,22 @@ namespace HighTempRadiantSystem {
         Real64 TimeStepSysSec = state.dataHVACGlobal->TimeStepSysSec;
```

Comment:
This would help here:  auto &thisHTR = state.dataHighTempRadSys->HighTempRadSys(RadSysNum);

Reply:
I'll definitely go back and add that!

Reply:
Implemented this in a variety of subroutines in this module that hadn't used it before.  Much cleaner now.  Will show up in the next commit.

Reply:
Did an intermediate commit specifically for this change so as to keep change easy to separate if something goes awry.